pkg/kubernetes/controllerutil.go

```diff
@@ -152,7 +153,7 @@ func mutate(f controllerutil.MutateFn, key client.ObjectKey, obj runtime.Object)
 }
 
 // LookupResource lookup for resource to be created by separate resource controller
-func LookupResource(name, namespace string, resource runtime.Object, client client.Client, logger logr.Logger) (*reconcile.Result, error) {
+func LookupResource1(name, namespace string, resource runtime.Object, client client.Client, logger logr.Logger) (*reconcile.Result, error) {
```

Comment:
Do we need function with number in name, i.e. `LookupResource1`

Reply:
no, this is non more needed